src/logic/denumerable.lean

```diff
@@ -280,7 +282,7 @@ namespace denumerable
 open encodable
 
 /-- An infinite encodable type is denumerable. -/
-def of_encodable_of_infinite (α : Type*) [encodable α] [infinite α] : denumerable α :=
+def of_encodable_of_infinite (α : Type*) [infinite α] [encodable α] : denumerable α :=
```

Comment:
LGTM, but why change the order here?

Reply:
It's to simplify the proof of `nonempty_denumerable` by not using `exactI`

Reply:
The name no longer matches the argument order. I think to reduce churn you should change it back.